src/components/toast/components/close-button/types.ts

```diff
@@ -0,0 +1,14 @@
+export interface CloseButtonProps {
+  /**
+   * A function that will be called when the button is clicked.
+   */
+  onClick?: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void
+  /**
+   * Optional class to be added to the button element.
+   */
+  className?: string
+  /**
+   * A label that describes the buttons action. Applied as aria-label value.
+   */
+  ariaLabel: string
```

Comment:
🎨 Can these be alphabetized and also have spaces between each for more separation?

I have the same request for:

- `src/components/toast/components/toast-display/types.ts`
- `toastOptions` currently defined in `src/components/toast/index.tsx`

```suggestion
  /**
   * A label that describes the buttons action. Applied as aria-label value.
   */
  ariaLabel: string

  /**
   * Optional class to be added to the button element.
   */
  className?: string

  /**
   * A function that will be called when the button is clicked.
   */
  onClick?: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void
```